test-network-function/networking/suite.go

```diff
@@ -275,3 +293,96 @@ func testNodePort(env *config.TestEnvironment) {
 		}
 	})
 }
+
+func getDeclaredPortList(container int, podName, podNamespace string) {
```

Comment:
I think that `declaredPort` should be the return value from this function versus setting a global variable.

Reply:
you're right, I fixed.